server/migrations/20210925191009-create-post.ts

```diff
@@ -0,0 +1,44 @@
+"use strict";
+module.exports = {
+  up: async (queryInterface: any, Sequelize: any) => {
+    await queryInterface.createTable("Posts", {
+      id: {
+        allowNull: false,
+        primaryKey: true,
+        type: Sequelize.STRING,
+      },
+      title: {
+        type: Sequelize.TEXT,
+      },
+      body: {
+        type: Sequelize.STRING,
+      },
+      thumbnail: {
+        type: Sequelize.STRING,
+      },
+      tags: {
+        type: Sequelize.STRING,
+      },
+      location: {
```

Comment:
City? Country? 

It seems like the intention here is a front end field "Location". Lets be more specific what this means. Lets also figure out if we need City/Country or just one?

Reply:
The location needs to work with google maps API which is something we'll implement later.. for now it is just a placeholder. We will likely store coordinates.

Reply:
Fair enough. It may not make sense to use coordinates though since they're not human readable. You may want to store the full address in the db.